op-challenger/flags/flags.go

```diff
@@ -37,10 +37,10 @@ var (
 		Usage:   "Address of the fault game factory contract.",
 		EnvVars: prefixEnvVars("GAME_FACTORY_ADDRESS"),
 	}
-	GameAddressFlag = &cli.StringFlag{
-		Name:    "game-address",
-		Usage:   "Address of the Fault Game contract.",
-		EnvVars: prefixEnvVars("GAME_ADDRESS"),
+	GameAllowlistFlag = &cli.StringSliceFlag{
+		Name:    "game-allowlist",
+		Usage:   "List of Fault Game contract addresses the challenger is allowed to play.",
```

Comment:
Should leave a comment to the effect that the challenger will play all games if the allowlist is empty.